mangaki/mangaki/management/commands/compare.py

```diff
@@ -135,16 +136,23 @@ def compare_models(self, nb_split: int, full_cv: bool):
         metrics = defaultdict(lambda: defaultdict(list))
 
         for pass_index, (i_train, i_test) in enumerate(k_fold.split(self.anonymized.X), start=1):
+            X_train = self.anonymized.X[i_train]
+            y_train = self.anonymized.y[i_train]
+            X_test = self.anonymized.X[i_test]
+            y_test = self.anonymized.y[i_test]
+            # pd.DataFrame(np.column_stack((X_train, y_train))).to_csv('/tmp/train.dat', header=False, index=False)
```

Comment:
should be deleted.

Reply:
Mais j'ai pas envie :o C'est utile……… un peu.

Reply:
Alors écris un flag pour exécuter ce code conditionnellement.